Makefile

```diff
@@ -1,12 +1,16 @@
  
-.PHONY: clean clean-mir ui-mir
+.PHONY: clean clean-mir default ui-mir
+
+default: ui-mir
 
 clean: clean-mir
 
+RUSTC:=rustc
+RUSTC_OPTIONS=-C overflow-checks=off -Zmir-enable-passes=-ConstDebugInfo
```

Comment:
@yanliu18 Do you feel these flags are sufficent? I notice in handwritten tests there are more